src/app/item-popup/PlugTooltip.tsx

```diff
@@ -150,12 +150,13 @@ export function PlugTooltip({
               <ClarityDescriptions
                 hash={def.hash}
                 fallback={<RichDestinyText text={perkDesc.description || perkDesc.requirement} />}
+                comunityOnly={showCommunityDescriptionOnly}
               />
             )
           )}
         </div>
       ))}
-      {sourceString && <div>{sourceString}</div>}
+      {!showCommunityDescriptionOnly && sourceString && <div>{sourceString}</div>}
```

Comment:
Why remove this?

Reply:
Fixed I intended to move it not remove it